lizard_efcis/views.py

```diff
@@ -480,3 +492,44 @@ def _key(point):
             lines.append(line)
 
         return Response(lines)
+
+
+class BoxplotAPI(FilteredOpnamesAPIView):
```

Comment:
Idem, meervoud: BoxplotsAPI, je geeft er meerdere terug.

Naam enzo ('efcis-boxplots') is al wel OK.